Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM docker.io/golang:bookworm AS builder
+
+WORKDIR src
+
+COPY *.go ./
```

Comment:
`go.mod` and `go.sum` are missing.

This will allow to remove the `go mod` commands below.

Reply:
Thanks for the review. However I'm a little reluctant to work on this as this PR has been open for a while and I've not received any feedback from the repo owner.